cpp/ql/lib/semmle/code/cpp/dataflow/internal/DataFlowImpl.qll

```diff
@@ -29,7 +29,12 @@ signature module FullStateConfigSig {
   /**
    * Holds if `sink` is a relevant data flow sink accepting `state`.
    */
-  predicate isSink(Node sink, FlowState state);
+  predicate isSinkWithState(Node sink, FlowState state);
```

Comment:
We want the FullStateConfigSig to match StateConfigSig with the only difference being whether or not some predicates have defaults.

Reply:
Should be fixed in 9a0a4a3611e7001624908f3dd1044fa3b4febb60 🤞.